File: src/main/Java/boj_step5/boj1152.java
import java.io.*;
import java.util.StringTokenizer;

public class boj1152 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        writer.append(String.valueOf(tokenizer.countTokens()));
        writer.close();
    }
}

File: src/main/kotlin/boj1152.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import java.util.StringTokenizer

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val s = r.readLine()
    var st = StringTokenizer(s)
    val count = st.countTokens()
    w.write(count.toString())
    w.close()
}